Composer/yarn.lock

```diff
@@ -3150,6 +3150,16 @@
     react-lifecycles-compat "^3.0.4"
     warning "^3.0.0"
 
+"@reach/router@^1.3.4":
```

Comment:
Are we changing our routing to use this over react-router?

Reply:
When I started working in the Composer code base the set router was already Reach/router. 

When choosing a routing library to help with navigation in my UI plug in I chose reach/router to stay consistent with the routing libs used in the larger codebase.